src/Arcus.Templates.ServiceBus.Queue/Program.cs

```diff
@@ -87,6 +87,8 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                 .MinimumLevel.Debug()
                 .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
                 .Enrich.FromLogContext()
+                .Enrich.WithVersion()
```

Comment:
We should add correlation as well or is this pending a release?

Reply:
No, correlation in message pumps are on message-level so are not configured here. They are still not yet released though, so waiting for the next version of Arcus.Messaging.
See this line: https://github.com/arcus-azure/arcus.messaging/blob/37703b956c6f950bcec6e658a3bdf2b1f28e71a9/src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs#L450